test/unit/runner_test.rb

```diff
@@ -3,9 +3,20 @@
 require "helper"
 require "inspec/secrets"
 require "inspec/runner"
+require "fetchers/mock"
 
 describe Inspec::Runner do
-  let(:runner) { Inspec::Runner.new({ command_runner: :generic }) }
+  let(:runner) { Inspec::Runner.new({ command_runner: :generic, reporter: [] }) }
+
+  it "bug #4524" do
+    file = 'describe "x" do before(:all) { command("y") }; it "z" do end; end'
+    runner.add_target("bug4524.rb" => file)
```

Comment:
Think you forgot to add this file?

Reply:
nope. it's the "mock" way of fetching. Totally undocumented but that's a separate issue.